scripts/cli_options.py

```diff
@@ -398,7 +418,7 @@ def get_sendpayment_parser():
         type='int',
         dest='makercount',
         help='how many makers to coinjoin with, default random from 4 to 6',
-        default=random.randint(4, 6))
+        default=random.randint(8, 10))
```

Comment:
Similar comments to earlier. This will lead to more failures and higher fees. I'd be more inclined to leave this one a bit lower tbh.

One thing to bear in mind is the current user experience with sendpayment is: it's a real time usage, but fairly slow (a minute is typical for the whole thing when working well), and if a failure happens on the first pass through, it actually hangs, waiting for 20*maker_timeout_sec before retrying. Users can (and generally, will), just ctrl-C out of it.